plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java

```diff
@@ -107,6 +111,107 @@ public void findEffectivePluginsTest() {
         assertEquals("1.0.1", effectivePlugins.get("ace-editor").getVersion().toString());
     }
 
+    @Test
+    public void listPluginsNoOutputTest() throws IOException {
+        Config config = Config.builder()
+                .withJenkinsWar(Settings.DEFAULT_WAR)
+                .withPluginDir(Files.createTempDirectory("plugins").toFile())
+                .withShowPluginsToBeDownloaded(false)
+                .build();
+
+        PluginManager pluginManager = new PluginManager(config);
+
+        ByteArrayOutputStream expectedNoOutput = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(expectedNoOutput));
+
+        assertEquals("", expectedNoOutput.toString().trim());
+    }
+
+    /*
```

Comment:
?

Reply:
It's currently commented out because it's not working. For some reason the value returned/output to the console is empty.